src/main/java/Qchat.java

```diff
@@ -0,0 +1,62 @@
+import javax.swing.*;
+import java.util.Scanner;
+
+public class Qchat {
+
+   static final String  WELCOME_GREETING = new String(
+           "Qchat ,A truly humanized intelligent voice assisstant \n"
+                   +"knows better about life and better about you\n"
+                   +"What can I do for you?\n"
+                   +"----------------------------------------------------------------\n") ;
+   static final String GOODBYE_GREETING = new String(
+           "--------------------------------------------------------\n" +
+            "goodbye\n"+ "Qchat, your life-long trusted companion\n");
+
+
+
+
+   static String CommandReader(){
+
+       Scanner in = new Scanner(System.in);
+       String command ;
+       command = in.nextLine();
+
+       switch (command){
+       case "Bye":
+           System.out.print(GOODBYE_GREETING);
+           break;
+       case "list":
+           Todolist.HandleList();
+           break;
+
+
+       default:
+           echo(command);
+           break;
+
+       }
```

Comment:
Why is the other commands such as quit, help, mark, unmark etc. only accessible after the user input list?